Adyen/Core/Components/Installment/InstallmentOptions.swift

```diff
@@ -4,19 +4,23 @@
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
 //
 
-@_spi(AdyenInternal) import Adyen
 import Foundation
 
+@_spi(AdyenInternal)
+public protocol InstallmentConfigurationAware {
+    var installmentConfiguration: InstallmentConfiguration? { get }
+}
+
 /// Details to configure Installment Options.
 /// These always include regular monthly installments,
 /// but in some countries `revolving` option may be added.
-public struct InstallmentOptions: Equatable {
+public struct InstallmentOptions: Equatable, Decodable, Encodable {
     
     /// Month options for regular installments.
-    internal let regularInstallmentMonths: [UInt]
+    public let regularInstallmentMonths: [UInt]
     
     /// Determines if revolving installment is an option.
-    internal let includesRevolving: Bool
+    public let includesRevolving: Bool
```

Comment:
maybe mark them with `@_spi(AdyenInternal)`?

Reply:
I realized hiding these serves no purpose and exposing them makes more sense if anybody wanted to check their config objects etc.

Reply:
not sure about that, we can always expose them later, but we can't make them private after exposing them, so IMHO its best if we make them spi_internal until merchants need them